tests/factories/models.py

```diff
@@ -35,6 +39,20 @@ class Meta:
     email = factory.SubFactory(factory="tests.factories.models.EmailFactory")
 
 
+class FirefoxAccountFactory(BaseSQLAlchemyModelFactory):
+    class Meta:
+        model = models.FirefoxAccount
+
+    fxa_id = factory.LazyFunction(lambda: uuid4().hex)
+    primary_email = factory.SelfAttribute("email.primary_email")
+    created_date = factory.Faker("date")
+    lang = factory.Faker("language_code")
+    first_service = factory.Faker("random_element", elements=["Firefox"])
+    account_deleted = factory.Faker("boolean", chance_of_getting_true=30)
```

Comment:
What's the value of not setting it always to False?

Reply:
There is none -- bad copypasta 🤦🏻‍♂️. [34ac959](https://github.com/mozilla-it/ctms-api/pull/634/commits/34ac9595c94eab3a1639d39562e776106917e26a)